mangaki/mangaki/templates/works_mosaic.html

```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-xs-12 cards-grid">
+    {% for slot_and_work in works %}
```

Comment:
If we get rid of the tuple we can just do `{% for work_meta in works %}` in `works.html` then pass the `work=work_meta.work slot_type=work_meta.slot_type` in the `{% include %}` below. I believe Django uses `None` as the default for missing keys.